Modify issue labeler to skip linked issues

roaks3 opened the pull request:
<!-- Put a description of what this PR is for here, along with any references to issues that this resolves or contributes to -->

Context is in the comment: adding service labels to a linked issue creates ambiguity that is difficult for us to detect and address, so we decided it was better to skip them.

<!--
Please self-review your PR against the review checklist before creating it: https://googlecloudplatform.github.io/magic-modules/contribute/review-pr/

Completing the checklist will help speed up the review process, and we appreciate you spending time on them before sending
your code to be reviewed.

If your PR is still work in progress, please create it in draft mode
-->

<!-- AUTOCHANGELOG for Downstream PRs.

Please select one of the following "release-note:" headings:
    - release-note:enhancement
    - release-note:bug
    - release-note:note
    - release-note:new-resource
    - release-note:new-datasource
    - release-note:deprecation
    - release-note:breaking-change
    - release-note:none

Unless you choose release-note:none, please add a release note.

See https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/ for writing good release notes.

You can add more release note blocks if you want more than one CHANGELOG
entry for this PR.
-->
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

modular-magician commented:
Hi there, I'm the Modular magician. I've detected the following information about your changes:

## Diff report

Your PR hasn't generated any diffs, but I'll let you know if a future commit does.

melinath reviewed (changes requested):
could you add a test for this behavior?

melinath commented:
actually this seems like it should have caused the "add missing service labels even if already linked" to start failing... but apparently it didn't? Are we running these tests automatically?

roaks3 commented:
Ah yep, that thought crossed my mind here but I forgot to follow up. Let me look at added a test for this.

roaks3 commented:
Confirmed that the test was not run in CI, and it did fail when running locally. I've updated the test to reflect the change, but we should update them to run in PRs.

cc @trodge is this a known limitation, or perhaps we removed this behavior by accident?

modular-magician commented:
Hi there, I'm the Modular magician. I've detected the following information about your changes:

## Diff report

Your PR hasn't generated any diffs, but I'll let you know if a future commit does.

melinath reviewed (approved):
LGTM. It would be great to also run tests on PRs as a GHA - but that doesn't need to happen in this PR.

trodge commented:
> Confirmed that the test was not run in CI, and it did fail when running locally. I've updated the test to reflect the change, but we should update them to run in PRs.
> 
> cc @trodge is this a known limitation, or perhaps we removed this behavior by accident?

I don't think we ever ran these tests as part of CI.

roaks3 commented:
Created https://github.com/hashicorp/terraform-provider-google/issues/17598
